superset/connectors/sqla/models.py

```diff
@@ -1087,7 +1096,6 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
             metrics_exprs = []
 
         time_range_endpoints = extras.get("time_range_endpoints")
-        groupby_exprs_with_timestamp = OrderedDict(groupby_exprs_sans_timestamp.items())
```

Comment:
`OrderedDict` is no longer necessary, as `dict` is guaranteed to be ordered as of Python 3.7.

![image](https://user-images.githubusercontent.com/33317356/132850155-ebb623fe-c19f-4df6-8085-3b4dd03a4505.png)

https://docs.python.org/3.8/library/stdtypes.html